doc/trex_vm_manual.asciidoc

```diff
@@ -382,11 +382,11 @@ zmq publisher at: tcp://*:4500
 <1> Output trimmed.
 
 [NOTE]
-See http://trex-tgn.cisco.com/trex/doc/trex_book.pdf[TRex full manual] for a complete understading of the tool features and options.
+See link:https://trex-tgn.cisco.com/trex/doc/trex_manual.html[TRex full manual] for a complete understading of the tool features and options.
```

Comment:
This better be relative 

```
link:trex_manual.html[TRex full manual]
```
please convert all the other links to relative

Reply:
Sorry for waiting so long.